qualtran/bloqs/multiplexers/select_pauli_lcu.py

```diff
@@ -142,6 +142,9 @@ def get_ctrl_system(self, ctrl_spec: 'CtrlSpec') -> Tuple['Bloq', 'AddControlled
     def adjoint(self) -> 'Bloq':
         return self
 
+    def _has_unitary_(self):
```

Comment:
Why is this necessary in this PR?

Reply:
`cirq.has_unitary` fails to realize that this is unitary because when it decomposes, it gets `And^dagger` gates which aren't unitary (#1488)